packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { cloneElement } from 'react';
+import styled, { css } from 'styled-components';
+import { shape, string, bool, func, oneOf } from 'prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { navigationIcons } from '@bbc/psammead-assets/svgs';
+import { C_WHITE, C_EBON, C_SHADOW } from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING_HLF,
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+import Helmet from 'react-helmet';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const MENU_ICON_SIDE_LENGTH = '2.75rem'; // 44px
+
+const getStyles = dir => {
+  const direction = dir === 'ltr' ? 'left' : 'right';
+  return `border-${direction}: ${GEL_SPACING_HLF} solid ${C_WHITE};
+          padding-${direction}: ${GEL_SPACING};`;
+};
+
+export const Dropdown = styled.div`
+  background-color: ${C_EBON};
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    display: none;
+    visibility: hidden;
+  }
+`;
+
+export const DropdownUl = styled.ul.attrs({ role: 'list' })`
+  list-style-type: none;
+  margin: 0;
+  padding: 0 ${GEL_SPACING};
+  border-bottom: 0.125rem solid ${C_SHADOW};
+`;
+
+const StyledDropdownLi = styled.li`
+  padding: 0.75rem 0;
+  border-bottom: 0.0625rem solid ${C_SHADOW};
+
+  &:last-child {
+    padding-bottom: ${GEL_SPACING_DBL};
+    border: 0;
+  }
+`;
+
+const StyledDropdownLink = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => service && getSansRegular(service)}
+  color: ${C_WHITE};
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+
+const StyledCurrentLink = styled.span`
+  ${({ dir }) => getStyles(dir)}
+`;
+
+export const DropdownLi = ({
+  children,
+  script,
+  currentPageText,
+  active,
+  service,
+  url,
+  dir,
+}) => (
+  <StyledDropdownLi role="listitem">
+    <StyledDropdownLink script={script} service={service} href={url}>
+      {active && currentPageText ? (
+        <StyledCurrentLink
+          dir={dir}
+          // eslint-disable-next-line jsx-a11y/aria-role
+          role="text"
```

Comment:
we could use styled component's `attrs` function to add the aria-role, like [here](https://github.com/bbc/psammead/blob/b33d9e8f3b56a7c0ae5b3eb5e13052bbdb5180cd/packages/components/psammead-copyright/src/index.jsx#L7-L9). This then means we won't have to have that ugly comment here. Just a suggestion though

Reply:
Good point, will add that 👍